tox.ini

```diff
@@ -67,5 +67,4 @@ commands =
     rm tests/__init__.py
     pycodestyle enterprise tests test_utils
     pydocstyle enterprise tests test_utils
-    isort --check-only --recursive tests enterprise test_utils manage.py setup.py test_settings.py
```

Comment:
isort wants imports in try-except blocks in the last position, whereas pylint (more correctly, I think) keeps them in the standard stdlib-external-internal position. I may have found a way to flag this to get isort to ignore it, though.